packages/react-charting/UnitTests/LineChartUT.test.tsx

```diff
@@ -0,0 +1,533 @@
+        },
+        {
+          x: 7,
+          y: 80,
+        },
+        {
+          x: 8,
+          y: 90,
+        },
+      ],
+      color: DefaultPalette.blue,
+      lineOptions: {
+        lineBorderWidth: '4',
+      },
+    },
+    {
+      legend: 'All',
+      data: [
+        {
+          x: 0,
+          y: 100,
+        },
+        {
+          x: 1,
+          y: 200,
+        },
+        {
+          x: 2,
+          y: 300,
+        },
+        {
+          x: 3,
+          y: 400,
+        },
+        {
+          x: 4,
+          y: 500,
+        },
+        {
+          x: 5,
+          y: 600,
+        },
+        {
+          x: 6,
+          y: 700,
+        },
+      ],
+      color: DefaultPalette.green,
+      lineOptions: {
+        lineBorderWidth: '4',
+      },
+    },
+    {
+      legend: 'single point',
+      data: [
+        {
+          x: 0,
+          y: 100,
+        },
+      ],
+      color: DefaultPalette.yellow,
+    },
+    {
+      legend: 'two points',
+      data: [
+        {
+          x: 0,
+          y: 100,
+        },
+        {
+          x: 1,
+          y: 200,
+        },
+      ],
+      color: DefaultPalette.green,
+    },
+    {
+      legend: 'three points',
+      data: [
+        {
+          x: 0,
+          y: 100,
+        },
+        {
+          x: 1,
+          y: 200,
+        },
+        {
+          x: 2,
+          y: 300,
+        },
+      ],
+      color: DefaultPalette.blue,
+    },
+    {
+      legend: 'four points',
+      data: [
+        {
+          x: 0,
+          y: 100,
+        },
+        {
+          x: 1,
+          y: 200,
+        },
+        {
+          x: 2,
+          y: 300,
+        },
+        {
+          x: 3,
+          y: 400,
+        },
+      ],
+      color: DefaultPalette.blue,
+    },
+  ],
+};
+
+runTest('Unit tests for _getPath function', () => {
+  testWithoutWait(
+    'Should return correct path when multiple shapes for points are not allowed and widthRatio and w equal to 1',
+    LineChart,
+    { data: basicChartPoints, allowMultipleShapesForPoints: false, strokeWidth: 0.5 },
+    container => {
+      const svg = container.querySelector('svg');
+      expect(svg).not.toBeNull();
+      const paths = svg!.querySelectorAll('path');
+      expect(paths).toHaveLength(6);
+      expect(paths[0]!.getAttribute('d')).toBe(`M-20.5 -17.065217391304344
+     A0.5 0.5 0 1 0 -19.5 -17.065217391304344
+     M-20.5 -17.065217391304344
+     A 0.5 0.5 0 1 1 -19.5 -17.065217391304344
+     `);
+      expect(paths[1]!.getAttribute('d')).toBe(`M15.5 12.826086956521738
+     A0.5 0.5 0 1 0 16.5 12.826086956521738
+     M15.5 12.826086956521738
+     A 0.5 0.5 0 1 1 16.5 12.826086956521738
+     `);
+      expect(paths[2]!.getAttribute('d')).toBe(`M27.5 0.8695652173913064
+     A0.5 0.5 0 1 0 28.5 0.8695652173913064
+     M27.5 0.8695652173913064
+     A 0.5 0.5 0 1 1 28.5 0.8695652173913064
+     `);
+    },
+  );
+
+  testWithWait(
```

Comment:
no change needed from fat arrow notation to normal function call?

Reply:
no, it is needed when we are spying on functions in the base class

Reply:
so those tests are not part of this PR?

Reply:
nope, here in unit tests we are not spying on any function